Content.Server/Kitchen/EntitySystems/ReagentGrinderSystem.cs

```diff
@@ -275,12 +283,17 @@ private void DoWork(ReagentGrinderComponent component, IEntity user, SharedReage
                         {
                             if (!item.HasTag("Grindable")) continue;
                             if (!item.TryGetComponent<SolutionContainerComponent>(out var solution)) continue;
-                            if (component.HeldBeaker.CurrentVolume + solution.CurrentVolume > component.HeldBeaker.MaxVolume) continue;
+                            var juiceEvent = new JuiceableScalingEvent(); // default of scalar is always 1.0
+                            if (item.HasComponent<StackComponent>())
+                            {
+                                RaiseLocalEvent<JuiceableScalingEvent>(item.Uid, juiceEvent);
+                            }
```

Comment:
you don't need to do the HasComponent check; the effect args just won't be modified if there isn't a stackcomp

also, make sure that you're raising the event directed by passing in false for the third arg
```suggestion
                            RaiseLocalEvent<JuiceableScalingEvent>(item.Uid, juiceEvent, false);
```